modules/ROOT/pages/dataweave-formats-multipart.adoc

```diff
@@ -264,6 +264,168 @@ provided in the `Content-Disposition` header, and note that DataWeave can
 handle content from supported formats, as well as references to unsupported
 ones, such as HTML.
 
+The following example shows how to send the current payload as a file part in a multipart form-data. You need to use a DataWeave script similar to the following:
```

Comment:
Is the second sentence necessary here? "You need to use a DataWeave script similar to the following:" 
Because you are already clarifying that this is an example, maybe it can be removed.